ckanext/qgov/common/views/user.py

```diff
@@ -1,28 +1,16 @@
 # encoding: utf-8
 
-import six
+from six import text_type as str
 
 from flask import Blueprint
 
 import ckan.lib.helpers as h
-from ckan.plugins.toolkit import _, check_ckan_version, g, request, redirect_to, url_for
+from ckan.plugins.toolkit import _, g, request, redirect_to, url_for
 from ckan.views.user import login, me, EditView
 
 blueprint = Blueprint(u'user_overrides', __name__)
 
 
-def dashboard_override(offset=0):
```

Comment:
why is this not required anymore? and you cross link the code that now does this.